### DISCUSSION TITLE
OpenZeppelin ERC20 Token Not Found in Wallet/Portfolio

### DISCUSSION THREAD

**Discussion by okhaimie-dev** [OpenZeppelin/cairo-contracts#837] [Q&A]
When I deploy an ERC20 token with Starkli, it shows up as a contract
https://goerli.voyager.online/contract/0x01d864b4b886fe108ab9a22ede564757c7e9530e4b8db55d0ed853162dd7e0c4#readContract

I would like to know if there is a way to either write the contract or deploy it to make it show up like an actual Token. Something like this
![CleanShot 2023-11-29 at 20 53 04](https://github.com/OpenZeppelin/cairo-contracts/assets/57156589/b65f53b0-c02a-4079-950f-5f45a3b635bd)

**Comment by martriay** [comment]
you should get in touch with the Voyager team, we don't know how they index tokens nor how is your contract

**Reply by okhaimie-dev** [reply]
This is the contract
https://github.com/okhaimie-dev/starknet-by-example/blob/main/src/openzeppelin/erc20.cairo

But I will reach out to Voyager, to see if they can provide me with an answer. Thanks
